workflow/scripts/extract-strains-from-gisaid-provision.py

```diff
@@ -0,0 +1,78 @@
+import sys
+
+sys.stderr = open(snakemake.log[0], "w")
+
+import json
+from os.path import join, isfile
+
+import numpy as np
+import pandas as pd
+
+
+def extrace_strains_from_provision(
+    path_to_provision: str, path_to_strain_summary: str, path_to_strain_genomes: str
+):
+    # select strain genomes
+    provision = pd.DataFrame()
+    chunks = pd.read_json(path_to_provision, lines=True, chunksize=10000)
+    for i, chunk in enumerate(chunks):
+        print(f"Parsing chunk {i}", file=sys.stderr)
+        provision = provision.append(select_oldest_strains(chunk), ignore_index=True)
+    provision = select_oldest_strains(provision)
```

Comment:
The reading of the JSON (about 40GB) takes ~7 min with the chunking. If someone knows a faster way, please feel free to point it out.